src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,40 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.9.0
+
+* The gateway Pod has been renamed from `submariner` to `submariner-gateway`.
+* The Helm charts now support Submariner deployments using the Operator.
+* Broker creation is now managed by the Operator (and not `subctl`).
+* Each Submariner Pod now has its own service account with appropriate privileges.
+* For OpenShift, the Lighthouse CoreDNS server metrics are now included with the default CoreDNS metrics.
+* The `submariner_connections` metric is renamed to `submariner_requested_connections`.
+* For cases in which cross-cluster connectivity is provided without Submariner, `subctl` can now just deploy
+  Service Discovery via the `--components` flag.
+* Improved Service CIDR discovery for k3s deployments.
+* All Submariner metrics are now prefixed with `submariner_` in Prometheus.
+* With Globalnet deployments, Global IPs are now assigned to exported Services only. Previously, Globalnet annotated
+  every Service in the cluster, whether or not it was exported.
+* Lighthouse now allows a user to set name of CoreDNS custom ConfigMap.
+* The `strongswan` cable driver which was deprecated in v0.8.0 release is now removed.
+* Globalnet can now be specifically disabled for a cluster on join via the `--globalnet` flag.
+* The Lighthouse-specific API is now removed in favor of [Kubernetes Multicluster Services API](https://github.com/kubernetes-sigs/mcs-api/).
+* A new tool [`subctl validate`](https://submariner.io/operations/deployment/subctl/#validate) was added that detects issues with the
+  Submariner deployment that may prevent it from working properly.
+* `subctl` commands now check if the `subctl` version is compatible with the deployed Submariner version.
+* Multiple new make-clean targets, `make clean-generated`, `make clean-clusters`and `make clean`, were added.
+* Submariner now supports automatic NAT discovery.
```

Comment:
```suggestion
* Submariner now supports automatic NAT discovery. 
```
@mangelajo shall we include an additional message that - "user supplied nat configuration will be used as a fallback option?"

Reply:
It's an option, that will be described in the nat section onf the website. WIP, but I'm fine with adding your explanation here too :) sounds good to me.

Reply:
Added it.